### PR TITLE
Update instructions to flash Meadow on Windows

### PR DESCRIPTION
**Pull request by jorgedevs**

### CODE REVIEW

File: docs/Meadow/Getting_Started/Deploying_Meadow/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ When you receive your Meadow board, it will need to have the latest Meadow.OS up
 2. [macOS](#macos)
 3. [Linux (Debian, Ubuntu)](#linux-debian-ubuntu)
 
-## Windows
+## **Windows**
```
Comment (adrianstevens) [inline docs/Meadow/Getting_Started/Deploying_Meadow/index.md:14]: Do we need both the bold and the title markup?

File: docs/Meadow/Getting_Started/Deploying_Meadow/index.md [hunk 2]
```diff
@@ -39,14 +39,26 @@ From a console with admin rights, execute following command:
 meadow install dfu-util
 ```
 
-### Update USB driver for ST devices
+### Ensure Meadow drivers are correct before deploying Meadow.OS
```
Comment (adrianstevens) [inline docs/Meadow/Getting_Started/Deploying_Meadow/index.md:42]: Maybe this should be 

### Install Meadow USB drivers 

Short and specific ... the old title is in the weeds, the new title is trying to do a little too much work imo

**Review by adrianstevens — APPROVED** [review]

File: docs/Meadow/Getting_Started/Deploying_Meadow/index.md (hunk 1 above)
Comment (jorgedevs) [reply docs/Meadow/Getting_Started/Deploying_Meadow/index.md]: Forgot to revert that, done!

File: docs/Meadow/Getting_Started/Deploying_Meadow/index.md (hunk 2 above)
Comment (jorgedevs) [reply docs/Meadow/Getting_Started/Deploying_Meadow/index.md]: I like your title suggestion better, thanks.

**Pull request merged**